invisible_cities/reco/paolina_functions.py

```diff
@@ -176,31 +176,105 @@ def length(track: Graph) -> float:
     return length
 
 
-def energy_within_radius(distances : Dict[Voxel, Dict[Voxel, float]], radius : float) -> float:
+def voxel_energy_within_radius(distances : Dict[Voxel, float], radius : float) -> float:
```

Comment:
I would use this name for the functions. I think is clearer, and it would be consistent with other function names like:   voxels_within_radius(...).

```suggestion
def voxels_energy_within_radius(distances : Dict[Voxel, float], radius : float) -> float:
```

Reply:
Here I think that `voxel_energy` is a correct way to write it, but English native speakers can confirm.

Reply:
I have changed the name of the function to `energy_of_voxels_within_radius`, under reviewer's suggestion.